src/components/auth/Login.jsx

```diff
@@ -62,6 +63,13 @@ class Login extends Component {
                 Login
               </button>
             </div>
+            <div className="social-media-container">
+              <SocialAuthButton />
+            </div>
+            <div className="signup-alt">
+              { 'Don\'t have an account?' }
+              <a href="/signup"> Signup </a>
```

Comment:
This should be implemented using a `Link`.

Reply:
I used anchor tag because link was causing Luqman's test to fail.

Reply:
resolved.